variables.tf

```diff
@@ -1456,3 +1456,9 @@ variable "miniobkp_s3_bucket_name_override" {
   default     = null
   description = "The name of the existing S3 bucket to be loaded and used as the minio backup bucket"
 }
+
+variable "efs_storage_class_name" {
```

Comment:
If this variable is being used for both efs and fsx than I recommend using a more generic name.